service/src/main/java/com/ge/predix/acs/privilege/management/dao/AttributeTraversalException.java

```diff
@@ -0,0 +1,14 @@
+package com.ge.predix.acs.privilege.management.dao;
+
+public class AttributeTraversalException extends RuntimeException {
```

Comment:
How about AttributeResolutionException ?  Also, note that we are not exposing this to our users.